inst/using-the-dccvalidator-app-amp-ad.Rmd

```diff
@@ -38,7 +38,22 @@ Some portions of the app submit data to Synapse. This allows curators at Sage to
 troubleshoot issues if needed; no one outside the Sage curation team will be
 able to download the data.
 
-## Documentation upload
+# Instructions
+
+## General Process Overview
```

Comment:
I might suggest we mention explicitly that this section has the overall overview of the process and that the following sections cover each of these steps in more detail.

Reply:
I've added a blurb here. I'm not sure if it's not clear enough. If it's not, I could change it to a list, like:

> The following sections are covered:
> - **General Process Overview** an overview of the data contribution process.
> - **Documentation Upload** how to upload study and assay documentation.
> - **Data Validation: Metadata Requirements** requirements for the metadata and manifest files and where to find templates.
> ...

Not sure which would be better, a blurb or a table of contents style.